inc/admin.php

```diff
@@ -234,7 +234,7 @@ function enqueue_admin_scripts( $prefix = '', $force = false ) {
 		if ( $force || self::is_admin() ) {
 			// Media is required for row styles
 			wp_enqueue_media();
-			wp_enqueue_script( 'so-panels-admin', plugin_dir_url( __FILE__ ) . '../js/siteorigin-panels' . SITEORIGIN_PANELS_VERSION_SUFFIX . SITEORIGIN_PANELS_JS_SUFFIX . '.js', array(
+			wp_enqueue_script( 'so-panels-admin', siteorigin_panels_url( 'js/siteorigin-panels' ) . SITEORIGIN_PANELS_VERSION_SUFFIX . SITEORIGIN_PANELS_JS_SUFFIX . '.js', array(
```

Comment:
For this, it might be slightly neater to have everything passed to the `siteorigin_panels_url` function. So:

```
siteorigin_panels_url( 'js/siteorigin-panels/' . SITEORIGIN_PANELS_VERSION_SUFFIX . SITEORIGIN_PANELS_JS_SUFFIX . '.js' )
```